sdk/src/main/scala/io/horizen/account/mempool/MempoolMap.scala

```diff
@@ -291,6 +291,24 @@ class MempoolMap(
     txsMap
   }
 
+  def mempoolTransactionsMapFrom(executable: Boolean, requestedFrom: AddressProposition): TrieMap[SidechainTypes#SCP, TxByNonceMap] = {
```

Comment:
why do we return map here?
i have a feeling the we do something very inefficient.
Please, contact @saratnt

Reply:
Method updated, as reported in the `go-ethereum` implementation (https://github.com/ethereum/go-ethereum/blob/master/internal/ethapi/api.go#L190) the `txpool_contentFrom` has a different interface than the `txpool_content` method.
`txpool_contentFrom` expose the data grouping them by nonce and transaction details (and not address/nonce/transaction)